FirebaseAuth/Sources/Backend/FIRAuthRequestConfiguration.h

```diff
@@ -77,6 +82,19 @@ NS_ASSUME_NONNULL_BEGIN
                         heartbeatLogger:(nullable id<FIRHeartbeatLoggerProtocol>)heartbeatLogger
     NS_DESIGNATED_INITIALIZER;
```

Comment:
This can be deleted (and any remaining calls to this should be updated to use the initializer with both heartbeatLogger and appCheck).

Reply:
deleted